starport/services/network/networktypes/genesisinformation.go

```diff
@@ -77,59 +77,136 @@ func NewGenesisInformation(
 	vestingAccs []VestingAccount,
 	genVals []GenesisValidator,
 ) (gi GenesisInformation) {
-
-	// convert account arrays into maps
-	gi.GenesisAccounts = make(map[string]GenesisAccount)
-	for _, genAcc := range genAccs {
-		gi.GenesisAccounts[genAcc.Address] = genAcc
-	}
-	gi.VestingAccounts = make(map[string]VestingAccount)
-	for _, vestingAcc := range vestingAccs {
-		gi.VestingAccounts[vestingAcc.Address] = vestingAcc
-	}
-	gi.GenesisValidators = make(map[string]GenesisValidator)
-	for _, genVal := range genVals {
-		gi.GenesisValidators[genVal.Address] = genVal
-	}
+	gi.GenesisAccounts = make([]GenesisAccount, len(genAccs))
```

Comment:
```suggestion
	return GenesisInformation{
	    GenesisAccounts:  genAccs,
	    VestingAccounts: vestingAccs,
	    GenesisValidators: genVals,
	}
```

Can we do this for simplicity?